packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -78,7 +79,7 @@ class IOSValidator extends DoctorValidator {
     if (xcode.isInstalled) {
       xcodeStatus = ValidationType.installed;
 
-      messages.add(ValidationMessage('Xcode at ${xcode.xcodeSelectPath}'));
+      messages.add(ValidationMessage(errorMessages.iOSXcodeLocation(xcode.xcodeSelectPath)));
```

Comment:
And everywhere they are used:

s/errorMessages/strings

Reply:
Done.